NitroxModel/DataStructures/GameLogic/Entities/Metadata/EntityMetadata.cs

```diff
@@ -32,6 +32,8 @@ namespace NitroxModel.DataStructures.GameLogic.Entities.Metadata
     [ProtoInclude(270, typeof(PlayerMetadata))]
     [ProtoInclude(280, typeof(GhostMetadata))]
     [ProtoInclude(290, typeof(WaterParkCreatureMetadata))]
+    [ProtoInclude(300, typeof(BeaconMetadata))]
+    [ProtoInclude(310, typeof(FlareMetadata))]
```

Comment:
Proper indexing from 50+n when (as discussed way back on Discord)? :)

Reply:
Will do that in #2090 as it's more related to metadatas